File: Lab2FragmentPizza/app/src/main/java/com/example/lab2fragmentpizza/BottomFragment.java
package com.example.lab2fragmentpizza;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import android.widget.TextView;

import androidx.fragment.app.Fragment;


public class BottomFragment extends Fragment {

    // private TextView textView;

    public BottomFragment() {
        // Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_bottom, container, false);
    }

    public void modifyTextView(String link) {
        TextView textView = (TextView) getView().findViewById(R.id.text);
        textView.setText(link);
    }

}

File: Lab2FragmentPizza/app/src/main/java/com/example/lab2fragmentpizza/BottomFragment.kt
package com.example.lab2fragmentpizza

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

class BottomFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        val view = inflater.inflate(R.layout.fragment_bottom, container, false)
        return view
    }

    fun setText(newText : String) {
        val textView = view?.findViewById(R.id.textView) as TextView
        textView.text = newText
    }

}
